projects/js-packages/image-guide/.eslintrc.cjs

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I see this file seems to have been copied from Boost, which does a bunch of special stuff since it uses svelte, and had a lot of existing code written under a different ruleset when it was imported.

Since this package doesn't seem to use React either, we could probably just delete this file. The only things eslint complains about without it are some doc blocks that need cleaning up.